src/main/java/com/coinbase/exchange/api/websocketfeed/WebsocketFeed.java

```diff
@@ -2,22 +2,33 @@
 
 import com.coinbase.exchange.api.exchange.Signature;
 import com.coinbase.exchange.api.gui.orderbook.OrderBookView;
-import com.coinbase.exchange.api.websocketfeed.message.*;
+import com.coinbase.exchange.api.websocketfeed.message.HeartBeat;
```

Comment:
keep individual imports - revert the unnecessary indentation for the rest of this file